docs/source/api/arduino.rst

```diff
@@ -0,0 +1,122 @@
+Arduino API
+===========
+
+The `Arduino <https://store.arduino.cc/arduino-uno-rev3>`__
+provides a total of 18 pins for either digital input or output (labelled
+2 to 13 and A0 to A5), including 6 for analogue input (labelled A0 to
+A5).
+
+Accessing the Arduino
+---------------------
+
+The Arduino can be accessed using the ``arduino`` property of
+the ``Robot`` object.
+
+.. code:: python
+
+   my_arduino = r.arduino
+
+You can use the GPIO *(General Purpose Input/Output)* pins for anything,
+from microswitches to LEDs. GPIO is only available on pins 2 to 13 and A0 to A5
+because pins 0 and 1 are reserved for communication with the rest of our
+kit.
+
+Setting the pin mode
```

Comment:
This is still not in the right place (it's before the introduction to the concept of pin modes - the section entitled "Pin modes" below). Due to time constraints I'll just make a PR to show how I think this should look.

Reply:
--> #42